test/User-test.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from 'chai';
```

Comment:
Test file looks thorough and covers happy/sad paths. Thank you for your work on building this!